requirements-pinned.txt

```diff
@@ -1,6 +1,7 @@
 cffi==1.14.4              # via cryptography, pynacl
 colorama==0.4.4
-cryptography==3.3.1
+cryptography==3.3.2       ; python_version < "3"
```

Comment:
Dependabot will likely soon submit a PR to bump this version, which we will need to ignore. But I don't want to ignore cryptography bumps altogether and I don't think we can configure Dependabot to only bump cryptography for python_version >= "3".

The good news is, once we drop Py2 we don't have to worry about it anymore.

Reply:
Maybe we can add an inline comment here for maintainers to ignore Dependabot bumps in this line?